x-pack/metricbeat/module/mssql/performance/performance.go

```diff
@@ -65,6 +65,10 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 	var err error
 	var rows *sql.Rows
+	var BufferCacheHitRatio = "Buffer cache hit ratio"
+	var BufferCacheHitRatioBase = "Buffer cache hit ratio base"
+	var buffer_cache_hit_ratio int64
+	var buffer_cache_hit_ratio_base int64
```

Comment:
```suggestion
         var buffer_cache_hit_ratio, buffer_cache_hit_ratio_base int64
         bufferCacheHitRatio := "Buffer cache hit ratio"
	bufferCacheHitRatioBase := "Buffer cache hit ratio base"
	
```

Whenever initializing, use "short assignment" but if just declaring then use "var". Idiomatic Go.

See: https://go.dev/tour/basics/10